### ISSUE TITLE
Cross profile notifications work one way only

### ISSUE THREAD

**Issue by bdwidgk784** [GrapheneOS/os_issue_tracker#4230]
I am running a Pixel 6a with Android 15. I have a particular VOIP app that requires Google Play Services to fully function properly. 

When I install Play Services and the VOIP app on User Profile 2 for example, and then I go back to the Owner profile to do my daily activities, when the same app gets a message on User Profile 2, I do not get any kind of notification (from User Profile 2) when I'm on the Owner profile. 

However, when I uninstall the same app on User Profile 2 (along with uninstalling Play Services), and install the same exact app on Owner profile (along with installing Play Services on Owner), and when I go back to User Profile 2 to do some activities, when the same app gets a message on Owner profile I immediately get the notification (from Owner profile) on User Profile 2 that says "Notification from app for Owner." 
  
So basically, cross profile notifications work only from apps installed on my Owner profile. But apps installed on my User Profile do not send cross profile notifications to my Owner profile or any other User profiles. 

In my User Profile 2, "send notifications to current user" is enabled, and "unrestricted background usage" is enabled for both the app and Play Services. 

Not sure if it is a bug or something?
